### PR TITLE
RelNotes: Preview Runtime 154, Prerelease SDK 154, Stable Runtime 153

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered articles for review:

* **Preview Runtime 154.0.4258.9 (Sep. 10, 2026)**
   * `/webview2/release-notes/runtime/154.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/runtime/154?branch=pr-en-us-3923
   * External preview: 
   * Planned live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/runtime/154
   * New page.
   * Improved the lead-in about testing.

* **SDK 1.0.4255-prerelease, for Runtime 154 (Sep. 11, 2026)**
   * `/webview2/release-notes/sdk/1-0-4255-prerelease.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease?branch=pr-en-us-3923
   * External preview: 
   * Planned live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease
   * New page.

* **Runtime 153.0.4234.32 (Sep. 11, 2026)**
   * `/webview2/release-notes/runtime/153.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/runtime/153?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/runtime/153
   * Changed from Preview Runtime content to Stable Runtime content.
   * Updated version # and date.

---
### Incidental files closely related to incoming RelNotes

* linked from each Runtime page to corresp. WebPlat relnotes page, from earliest webplat = [133](https://learn.microsoft.com/en-us/microsoft-edge/web-platform/release-notes/133) through current.
   * eg: https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/runtime/153?branch=pr-en-us-3923

* **Release notes for the WebView2 Runtime**
   * `/webview2/release-notes/runtime/index.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/runtime/index?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/runtime/
   * Added link: **Preview Runtime 154.0.4258.9 (Sep. 10, 2026)**
   * Updated link text from **Preview Runtime 153.0.4234.6 (Aug. 28, 2026)** to **Runtime 153.0.4234.32 (Sep. 11, 2026)**

* **Release notes for WebView2 SDKs**
   * `/webview2/release-notes/sdk/index.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/index?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/
   * Added: **Preview Runtime 154.0.4258.9 (Sep. 10, 2026)**
   * Changed link text from **Preview Runtime 153.0.4234.6 (Aug. 28, 2026)** to **Runtime 153.0.4234.32 (Sep. 11, 2026)**.

* `toc.yml`
   * Added: **Preview Runtime 154.0.4258.9 (Sep. 10, 2026)**
   * Changed from **Preview Runtime 153.0.4234.6 (Aug. 28, 2026)** to **Runtime 153.0.4234.32 (Sep. 11, 2026)**.
   * Added **SDK 1.0.4255-prerelease, for Runtime 154 (Sep. 11, 2026)**
   * Added new toc bucket **Prerelease testing using preview channels of Microsoft Edge**.  Contains articles:
      * Prerelease testing using preview channels of Microsoft Edge
      * Switch to a preview channel to test upcoming APIs and features
      * Self-host by deploying preview channels

* `/webview2/release-notes/includes/sdk-prerelease-template.md`
   * Changelist:

* `/webview2/release-notes/includes/sdk-release-template.md`
   * Changelist:

---
### Incidental files not closely related to incoming RelNotes

* **Prerelease testing using preview channels of Microsoft Edge**
   * `/webview2/how-to/prerelease-testing.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing
   * Changed title from: **Prerelease testing using preview channels**

* **Switch to a preview channel to test upcoming APIs and features**
   * `/webview2/how-to/set-preview-channel.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel
   * Restored old title; changed from: **Test upcoming APIs and features**
   * Added local toc.

* **Release SDK 0.8.270, for Runtime 78 (Sep. 10, 2019)**
   * `/webview2/release-notes/sdk/0-8-270.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/0-8-270?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/0-8-270
   * Updated link text.

* **Self-host by deploying preview channels**
   * `/webview2/how-to/self-hosting.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting
   * Changelist:

* **Distribute your app and the WebView2 Runtime**
   * `/webview2/concepts/distribution.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/concepts/distribution
   * Changelist:

* **Overview of WebView2 APIs**
   * `/webview2/concepts/overview-features-apis.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis
   * Changelist:

* **Develop secure WebView2 apps**
   * `/webview2/concepts/security.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/security?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/concepts/security
   * Changelist:

* **Prerelease and Release SDKs for WebView2**
   * `/webview2/concepts/versioning.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/versioning?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/concepts/versioning
   * Changelist:

* **Automate and test WebView2 apps with Microsoft Edge WebDriver**
   * `/webview2/how-to/webdriver.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/webdriver?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/how-to/webdriver
   * Changelist:

* **WebView2 documentation** - WebView2 landing page
   * `/webview2/landing/index.yml`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/landing/index?branch=pr-en-us-3923
   * External preview: 
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/landing/
   * Changelist:

AB#63903469

### CODE REVIEW

File: microsoft-edge/webview2/release-notes/includes/runtime-preview-template.md
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/includes/runtime-preview-template.md:1]: revert changes from global SNR

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+   * [CoreWebView2Environment.CreateOrJoinClusterEnvironmentAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createorjoinclusterenvironmentasync?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
+   * [CoreWebView2Environment.GetClusterEnvironmentOptions Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.getclusterenvironmentoptions?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
+
+<!-- 5 -->
+* `CoreWebView2Frame` Class:
+   * [CoreWebView2Frame.LaunchingExternalUriScheme Event](/dotnet/api/microsoft.web.webview2.core.corewebview2frame.launchingexternalurischeme?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
+
+<!-- 6 -->
+* `CoreWebView2LaunchingExternalUriSchemeEventArgs` Class:
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.Handled Property](/dotnet/api/microsoft.web.webview2.core.corewebview2launchingexternalurischemeeventargs.handled?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true) _todo: script emitted this member for .net & win32 lists only; delete, or add to winrt tab?_
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: delete, or add to winrt tab?  CoreWebView2LaunchingExternalUriSchemeEventArgs.Handled

script emitted this member for .net & win32 lists only

https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease?branch=pr-en-us-3923

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+   * [CoreWebView2ClusterEnvironmentOptions.ClusterName Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#clustername)
+   * [CoreWebView2ClusterEnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#customschemeregistrations)
+   * [CoreWebView2ClusterEnvironmentOptions.EnableTrackingPrevention Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#enabletrackingprevention)
+   * [CoreWebView2ClusterEnvironmentOptions.Handled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#handled) - _todo: script emitted for rt only; delete, or add to .net & win32?_
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: delete, or add to .net & win32?  CoreWebView2ClusterEnvironmentOptions.Handled

script emitted the api for winrt only

https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease?branch=pr-en-us-3923

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+   * [CoreWebView2Environment.GetClusterEnvironmentOptions Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#getclusterenvironmentoptions)
+
+<!-- 5 -->
+* `CoreWebView2Frame` Class:
+   * [CoreWebView2Frame.LaunchingExternalUriScheme Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#launchingexternalurischeme)
+
+<!-- 6 -->
+* `CoreWebView2LaunchingExternalUriSchemeEventArgs` Class:
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.IAsyncOperation Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#iasyncoperation)  _todo: script emitted this member for winrt list only; delete, or add to .net & win32 tabs?_
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: rt: IAsyncOperation: delete, or add to .net & win32 tabs?CoreWebView2LaunchingExternalUriSchemeEventArgs.IAsyncOperation

script emitted this member for winrt list only

https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease?branch=pr-en-us-3923

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+   * [ICoreWebView2ExperimentalClusterEnvironmentOptions::put_PerHostProfileIsolation](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclusterenvironmentoptions?view=webview2-1.0.4255-prerelease&preserve-view=true#put_perhostprofileisolation)
+   * [ICoreWebView2ExperimentalClusterEnvironmentOptions::put_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclusterenvironmentoptions?view=webview2-1.0.4255-prerelease&preserve-view=true#put_releasechannels)
+   * [ICoreWebView2ExperimentalClusterEnvironmentOptions::SetCustomSchemeRegistrations](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclusterenvironmentoptions?view=webview2-1.0.4255-prerelease&preserve-view=true#setcustomschemeregistrations)
+
+<!-- win32-only -->
+* [ICoreWebView2ExperimentalCreateOrJoinClusterEnvironmentCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcreateorjoinclusterenvironmentcompletedhandler?view=webview2-1.0.4255-prerelease&preserve-view=true)
+
+<!-- 4 -->
+_todo: add # suffix? confirm iface name and method names_
+* `ICoreWebView2ExperimentalEnvironment`:
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: add # suffix? confirm iface name ICoreWebView2ExperimentalEnvironment

confirm method names for win32:
CreateOrJoinClusterEnvironmentAsync
GetClusterEnvironmentOptions

https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease?branch=pr-en-us-3923

File: microsoft-edge/webview2/release-notes/runtime/153.md [hunk 5]
```diff
@@ -1,25 +1,28 @@
...
 ms.author: msedgedevrel
 ms.topic: article
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 08/28/2026
+ms.date: 09/07/2026
 ---
-# Preview Runtime 153.0.4234.6 (Aug. 28, 2026)
-<!-- todo: when this runtime gets released as a stable runtime, remove "preview" from title, update version number, update date, update lead-in sentence; per /release-notes/includes/runtime-template.md -->
+# Runtime nnn.n.nnnn.nn (Mmm. dd, yyyy)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/runtime/153.md]: put version # in title & lead-in & toc, etc. globally

File: microsoft-edge/webview2/release-notes/runtime/154.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+update version number
+update date
+update lead-in sentence
+del See Also link [Prerelease testing using preview channels of Microsoft Edge]
+-->
+
+Release notes for Microsoft Edge WebView2 Preview Runtime, release date: Sep. 3, 2026.
+
+<!-- todo: remove paragraph when this runtime gets released as a stable runtime -->
+This is a Preview Runtime, for early testing; specifically, the Beta (rather than Dev or Canary) Runtime build.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/runtime/154.md]: condense para

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2Frame` Class:
+   * [CoreWebView2Frame.LaunchingExternalUriScheme Event](/dotnet/api/microsoft.web.webview2.core.corewebview2frame.launchingexternalurischeme?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
+
+* `CoreWebView2LaunchingExternalUriSchemeEventArgs` Class:
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.Handled Property](/dotnet/api/microsoft.web.webview2.core.corewebview2launchingexternalurischemeeventargs.handled?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true) _todo: script emitted this member for .net & win32 lists only; delete, or add to winrt tab?_<!-- bug: script emitted this member for .net & win32, not for winrt -->
```
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: keep it

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2Frame` Class:
+   * [CoreWebView2Frame.LaunchingExternalUriScheme Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#launchingexternalurischeme)
+
+* `CoreWebView2LaunchingExternalUriSchemeEventArgs` Class:
+   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.IAsyncOperation Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#iasyncoperation)  _todo: script emitted this member for winrt list only; delete, or add to .net & win32 tabs?_<!-- bug: script emitted this member for winrt, not for .net, not for win32 -->
```
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```suggestion
   * [CoreWebView2LaunchingExternalUriSchemeEventArgs.Handled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2launchingexternalurischemeeventargs?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#handled)
```

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+
+<!-- 3 -->
+* [CoreWebView2ClusterEnvironmentStatus Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2clusterenvironmentstatus?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
+   * `NotSupported`
+   * `OptionsMismatch`
+   * `Succeeded`
+
+<!-- 4 -->
+* `CoreWebView2Environment` Class:
+   * [CoreWebView2Environment.CreateOrJoinClusterEnvironmentAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createorjoinclusterenvironmentasync?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)<!-- bug: the script emitted this member for .net only, not for rt or win32 -->
```
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```suggestion
   * [CoreWebView2Environment.CreateOrJoinClusterEnvironmentAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createorjoinclusterenvironmentasync?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+   * [CoreWebView2Environment.GetClusterEnvironmentOptions Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.getclusterenvironmentoptions?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)<!-- bug: the script emitted this member for .net only, not for rt or win32 -->
```
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```suggestion
   * [CoreWebView2Environment.GetClusterEnvironmentOptions Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.getclusterenvironmentoptions?view=webview2-dotnet-1.0.4255-prerelease&preserve-view=true)
```

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 11]
```diff
@@ -0,0 +1,213 @@
...
+   * [CoreWebView2ClusterEnvironmentOptions.ClusterName Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#clustername)
+   * [CoreWebView2ClusterEnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#customschemeregistrations)
+   * [CoreWebView2ClusterEnvironmentOptions.EnableTrackingPrevention Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#enabletrackingprevention)
+   * [CoreWebView2ClusterEnvironmentOptions.Handled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2clusterenvironmentoptions?view=webview2-winrt-1.0.4255-prerelease&preserve-view=true#handled) - _todo: script emitted for rt only; delete, or add to .net & win32?_<!-- bug: script emitted this member for winrt, not for .net, not for win32 -->
```
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```suggestion
```

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 12]
```diff
@@ -0,0 +1,213 @@
...
+* [ICoreWebView2ExperimentalCreateOrJoinClusterEnvironmentCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcreateorjoinclusterenvironmentcompletedhandler?view=webview2-1.0.4255-prerelease&preserve-view=true)
+
+<!-- 4 -->
+_todo: add # suffix? confirm iface name and method names_
+* `ICoreWebView2ExperimentalEnvironment`:
+   * [ICoreWebView2ExperimentalEnvironment::CreateOrJoinClusterEnvironmentAsync](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironment?view=webview2-1.0.4255-prerelease&preserve-view=true#createorjoinclusterenvironmentasync)
+   * [ICoreWebView2ExperimentalEnvironment::GetClusterEnvironmentOptions](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironment?view=webview2-1.0.4255-prerelease&preserve-view=true#getclusterenvironmentoptions)
+
+<!-- 3 -->
+* [COREWEBVIEW2_CLUSTER_ENVIRONMENT_STATUS Enum](/microsoft-edge/webview2/reference/win32/webview2-idl-experimental?view=webview2-1.0.4255-prerelease&preserve-view=true#corewebview2_cluster_environment_status)<!-- bug: script emitted webview2-idl in url, but should emit webview2-idl-experimental -->
```
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```suggestion
* [COREWEBVIEW2_CLUSTER_ENVIRONMENT_STATUS Enum](/microsoft-edge/webview2/reference/win32/webview2-idl-experimental?view=webview2-1.0.4255-prerelease&preserve-view=true#corewebview2_cluster_environment_status)
```

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md (hunk 4 above)
Comment (ojasvi1450) [reply microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: Keep something like this. 
```
<!-- 4 -->
* [CreateOrJoinCoreWebView2ClusterEnvironment](/microsoft-edge/webview2/reference/win32/webview2-idl-experimental?view=webview2-1.0.4255-prerelease&preserve-view=true#createorjoincorewebview2clusterenvironment)
* [GetCoreWebView2ClusterEnvironmentOptions](/microsoft-edge/webview2/reference/win32/webview2-idl-experimental?view=webview2-1.0.4255-prerelease&preserve-view=true#getcorewebview2clusterenvironmentoptions)
```

File: microsoft-edge/toc.yml [hunk 13]
```diff
@@ -2391,12 +2397,17 @@
             - name: Automate and test WebView2 apps with Microsoft Edge WebDriver
               href: ./webview2/how-to/webdriver.md
 
-            - name: Prerelease testing using preview channels
-              href: ./webview2/how-to/prerelease-testing.md
+            - name: Prerelease testing using preview channels of Microsoft Edge
+              items:
+                - name: Prerelease testing using preview channels of Microsoft Edge  # short article
```
Comment (mikehoffms) [inline microsoft-edge/toc.yml]: del two # comments

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```
<!-- 4 -->
_todo: add # suffix? confirm iface name and method names_
* `ICoreWebView2ExperimentalEnvironment`:
   * [ICoreWebView2ExperimentalEnvironment::CreateOrJoinCoreWebView2ClusterEnvironment](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironment?view=webview2-1.0.4255-prerelease&preserve-view=true#createorjoincorewebview2clusterenvironment)
   * [ICoreWebView2ExperimentalEnvironment::GetCoreWebView2ClusterEnvironmentOptions](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironment?view=webview2-1.0.4255-prerelease&preserve-view=true#getcorewebview2clusterenvironmentoptions)
```
still probably needs # suffix on type
closing, will enter fresh comment after push commit

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md [hunk 14]
```diff
@@ -0,0 +1,212 @@
...
+   * [ICoreWebView2ExperimentalClusterEnvironmentOptions::put_PerHostProfileIsolation](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclusterenvironmentoptions?view=webview2-1.0.4255-prerelease&preserve-view=true#put_perhostprofileisolation)
+   * [ICoreWebView2ExperimentalClusterEnvironmentOptions::put_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclusterenvironmentoptions?view=webview2-1.0.4255-prerelease&preserve-view=true#put_releasechannels)
+   * [ICoreWebView2ExperimentalClusterEnvironmentOptions::SetCustomSchemeRegistrations](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclusterenvironmentoptions?view=webview2-1.0.4255-prerelease&preserve-view=true#setcustomschemeregistrations)
+
+<!-- win32-only -->
+* [ICoreWebView2ExperimentalCreateOrJoinClusterEnvironmentCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcreateorjoinclusterenvironmentcompletedhandler?view=webview2-1.0.4255-prerelease&preserve-view=true)
+
+<!-- 4 -->
+_todo: add # suffix? confirm iface name and method names_
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]: `ICoreWebView2ExperimentalEnvironment` - add suffix #?
confirm iface name and method names
`CreateOrJoinCoreWebView2ClusterEnvironment`
`GetCoreWebView2ClusterEnvironmentOptions`

https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease?branch=pr-en-us-3923&tabs=win32cpp#shared-cluster-environment-api

File: microsoft-edge/webview2/release-notes/runtime/153.md [hunk 15]
```diff
@@ -1,20 +1,22 @@
...
 ms.author: msedgedevrel
 ms.topic: article
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 08/28/2026
+ms.date: 09/07/2026
 ---
-# Preview Runtime 153.0.4234.6 (Aug. 28, 2026)
-<!-- todo: when this runtime gets released as a stable runtime, remove "preview" from title, update version number, update date, update lead-in sentence; per /release-notes/includes/runtime-template.md -->
+# Runtime 153.n.nnnn.nn (Mmm. dd, yyyy)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/runtime/153.md]: put actual version #, globally in repo

update date

File: microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md (hunk 14 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/sdk/1-0-4255-prerelease.md]:
```
<!-- 4 -->
* Global functions:
   * [CreateOrJoinCoreWebView2ClusterEnvironment](/microsoft-edge/webview2/reference/win32/webview2experimental-idl?view=webview2-1.0.4255-prerelease&preserve-view=true#createorjoincorewebview2clusterenvironment)
   * [GetCoreWebView2ClusterEnvironmentOptions](/microsoft-edge/webview2/reference/win32/webview2experimental-idl?view=webview2-1.0.4255-prerelease&preserve-view=true#getcorewebview2clusterenvironmentoptions)
```

File: microsoft-edge/webview2/release-notes/runtime/153.md
Comment (ojasvi1450) [inline microsoft-edge/webview2/release-notes/runtime/153.md]:
```suggestion

```

**Review by ojasvi1450 — APPROVED** [review]
LGTM - Dev

**Review by mikehoffms — COMMENTED** [review]
LGTM - Writer/Editor
